git-multimail/README

```diff
@@ -388,6 +394,18 @@ multimailhook.replyToRefchange
     - The value "none", in which case the Reply-To: field will be
       omitted.
 
+multimailhook.refFilterInclusionRegex
+multimailhook.refFilterExclusionRegex
+
+    Regular expressions that can be used to limit refs for which email
+    updates will be sent.  It is an error to specify both an inclusion
+    and an exclusion regex.  If an inclusion regex is specified, emails
+    will only be sent for refs which match this regex.  If an exclusion
+    regex is specified, emails will be sent for all refs except those
+    that match this regex (or that match a predefined regex specific
+    to the environment, such as "^refs/notes" for most environments and
+    "^refs/notes|^refs/changes" for the gerrit environment).
```

Comment:
Actually, neither of these variables is a set/list/whatever of regexes.  It's a single regex.  But some folks only know very basic regexes (e.g. only being aware of the three special characters: ., +, and *), and I wanted to describe this variable while simultaneously hinting how they could match multiple things -- without being overly verbose.  I figured those who knew better would just say "A '|'-separated list of regexes for matching any one of the regexes?  How cute; that's just the normal regex syntax for matching multiple different things, so it's really just one regex."

Clearly, my wording didn't have the intended effect.  However, I like the fact that it's just a regex. If we make it a space separated list or multiple config values, then it loses that quality.  If that's the way you want it, I'll implement it that way, but I personally much prefer that each of these variables is just a plain python regex.